scripts/zones/Behemoths_Dominion/mobs/Behemoth.lua

```diff
@@ -15,6 +15,16 @@ function onMobInitialize(mob)
 end;
 
 -----------------------------------
+-- onMobSpawn
+-----------------------------------
+
+function onMobSpawn(mob)
+    if (LandKingSystem_NQ > 0 or LandKingSystem_HQ > 0) then
+        GetNPCByID(17297459):setStatus(STATUS_DISAPPEAR);
```

Comment:
Would this be better as an "and" because of the oddball scenario of one being timed but the other being set force pop? Then again maybe requiring ppl to take down the opposite one before they force-spawn the other is better (dragons aery in particular gets kinda crowded with 2 dragons :wink: ) I'm not complaining either way, just a thought because back when I ran a server I did that and I'm not sure which people would want.